host/xtest/xtest_uuid_helpers.c

```diff
@@ -82,6 +82,26 @@ TEEC_Result xtest_uuid_from_str(TEEC_UUID *uuid, const char *s)
 	return res;
 }
 
+#define UUID_STR_LEN	37
+
+char *xtest_uuid_to_str(const TEEC_UUID *uuid)
```

Comment:
It's not obious from caller implementation that there is an allocation done here that caller shall free by its own means. I suggest to rename the function so we know it allocates the output buffer (e.g. `xtest_uuid_to_alloced_str()` or let caller do the allocation and pass a `char str[UUID_STR_LEN]` 2nd argument.